README.md

```diff
@@ -12,14 +12,15 @@ We encorage you to run this tool within a [CloudShell](https://aws.amazon.com/cl
     ```
 1. Execute it
     ```
-    python3 main.py <domain>
+    python3 main.py <domain> --workspace <your-workspace-name>
     ```
 
 1. Tool will proceed to create the following resources:
     - Create Identity
     - Create DKIM records in the hosted zone.
     - Create MAIL From using `bounce.${domain}`
     - Create MX record
+    - Create RuleSet, and Rule to deliver arriving email to Cerby (you need to add the --workspace flag to create these resources)
```

Comment:
if this is optional, I believe we should run it separately

Reply:
i set it optional 'casue we have a lot of documentation, videos, etc, that would required to be updated 🤔

Reply:
resolving as we already agreed on removing workspace argument, and use the domain name instead.